### ISSUE TITLE
modelcluster breaks with recent taggit upgrade to 1.3

### ISSUE THREAD

**Issue by nkuttler** [wagtail/django-modelcluster#124]
I have a page model that has tags, and a modeladmin for it. Simply creating an object and trying to save if fails.

```
  File "/lib/python3.7/site-packages/modelcluster/contrib/taggit.py", line 44, in add
    tag_objs = self._to_tag_model_instances(tags)

Exception Type: TypeError at /wtadmin/pages/add/wt/mypage/3/
Exception Value: _to_tag_model_instances() missing 1 required positional argument: 'tag_kwargs'
```

Please let me know if you need more info. See also https://github.com/jazzband/django-taggit/issues/675

**Comment by zerolab** [comment]
#123 aims to fix this. can you please test with that change?

**Comment by rgs258** [comment]
Note wagtail/wagtail#6063 as well

**Comment by gasman** [comment]
Fixed in #123
